packages/appstore/src/components/wallet-transfer/wallet-transfer.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import { AmountInput, TransferAccountSelector } from '@deriv/components';
+import { getDecimalPlaces } from '@deriv/shared';
+import { useStore, observer } from '@deriv/stores';
+import { localize, Localize } from '@deriv/translations';
+import { transfer_accounts } from './mock_accounts/mock_accounts';
+import type { TAccount } from './types';
+import './wallet-transfer.scss';
+
+const Divider = () => <div className='wallet-transfer__divider' />;
+
+const WalletTransfer = observer(() => {
+    const { ui } = useStore();
+    const { is_mobile } = ui;
+
+    const [from_account, setFromAccount] = React.useState<TAccount>(transfer_accounts.wallets[0]);
+    const [to_account, setToAccount] = React.useState<TAccount>();
+    const [from_amount, setFromAmount] = React.useState();
+    const [to_amount, setToAmount] = React.useState();
```

Comment:
@heorhi-deriv These two states are unused 🤔

Reply:
@farzin-deriv for now yes, will use them in upcoming card related to inputs synchronization

Reply:
Alright, Ignore my comment please 🙏🏻 🙇🏻